### ISSUE TITLE
RedirectManager interface triggers an error because of wrong deprecated resource type

### ISSUE THREAD

**Issue by rdb987** [Adobe-Consulting-Services/acs-aem-commons#3413]
## Required Information

* [6.5.21 ] AEM Version, including Service Packs
* [6.6.2 ] ACS AEM Commons Version
* [yes ] Reproducible on Latest? yes

### Expected Behavior

When adding a redirect or editing a redirect rule in the redirect manager, redirect filter should be added and no error should be triggered

### Actual Behavior

Resource dumped by HtmlRendered is triggered and visible in the interface. 
The rows have following resource type: /apps/acs-commons/components/utilities/manage-redirects/redirect-row which now doesn't exist anymore at all in the JCR Repository.

### Steps to Reproduce

1. Open the redirect manager.
2. Add/modify whatever redirect rule
3. Click on Save
4. Error is reproduced.

### Links
![acs-commons-issue](https://github.com/user-attachments/assets/607115e8-2306-406a-bebc-56eeb036a043)

**Comment by rdb987** [comment]
Yes the refresh works and the redirects too but of course for our customer is not nice to see that error. Hope will be solved soon :)

**Comment by YegorKozlov** [comment]
fixed in v6.11
